### PR TITLE
Adding my_rhessys_package

### PR DESCRIPTION
**Pull request by DavidChoi76**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by DavidChoi76** [comment]
this PR is ready for review. thanks!

File: recipes/example/meta.yaml
Comment (synapticarbors) [inline recipes/example/meta.yaml:79]: please do not modify this file.

File: recipes/rhessys/meta.yaml
Comment (synapticarbors) [inline recipes/rhessys/meta.yaml:7]: We'd generally prefer for you to use a tarball release rather than a git revision:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

File: recipes/rhessys/LICENSE.txt
Comment (synapticarbors) [inline recipes/rhessys/LICENSE.txt:1]: You probably don't need to manually package the license file if it is in the source distribution you are using. Just point the entry in the `meta.yaml` to the location in the distribution directory.

**Review by synapticarbors — COMMENTED** [review]
These are just general comments and I'd probably want a reviewer with more experience with c/c++ builds to comment as well.

File: recipes/rhessys/meta.yaml
Comment (DavidChoi76) [reply recipes/rhessys/meta.yaml:7]: I used a tarball release.

File: recipes/rhessys/LICENSE.txt
Comment (DavidChoi76) [reply recipes/rhessys/LICENSE.txt]: I pointed the License file to `https://github.com/DavidChoi76/rhessysec/blob/master/LICENSE.txt` but that occurred an error. So I want to keep the License file in this folder.

**Comment by DavidChoi76** [comment]
This PR is ready for review again.

**Comment by DavidChoi76** [comment]
@synapticarbors In my case, It is the first time to create a conda package. So I don't know the exact procedure. Do I have to wait until you review my pull request again?

**Comment by DavidChoi76** [comment]
@synapticarbors I'm sorry to send a message again. I don't how long do I have to wait to merge my pull request in general.

File: recipes/example/meta.yaml
Comment (synapticarbors) [reply recipes/example/meta.yaml:79]: This file is still showing a diff.

File: recipes/rhessys/meta.yaml
Comment (synapticarbors) [reply recipes/rhessys/meta.yaml:7]: This still looks like it is pulling a git repo via `git_url` and the revision rather than  just using `url` with a source distribution tarball.

**Review by synapticarbors — COMMENTED** [review]
Can you get the co-maintainers to comment on this PR that they are willing to be listed?

Also, @conda-forge/help-c-cpp, I'd appreciate it if someone from the team could review this.

**Comment by laurencelin** [comment]
I am the developer of RHESSysEastCoast. I would like to be a co-maintainer for the rhessysec conda package.

**Comment by lband** [comment]
I am the developer of the original RHESSys code.  I would also like to be a co-mainainer for the rhessysec conda package.

**Comment by DavidChoi76** [comment]
@synapticarbors Maintainers of RHESSys conda package posted their comments on this PR. 
In your first comment, you said "This file is still showing a diff." But in your previous comment, you said "please do not modify this file." Do I need to do something about this?
In your second comment, you said "This still looks like it is pulling a git repo via git_url and the revision rather than just using url with a source distribution tarball." What should I do about this? My last commit for this PR is "https://github.com/conda-forge/staged-recipes/pull/13395/commits/8ecf344238164e7f07c6bdb8fe18086bf6bc7059", and I fixed "recipes/rhessys/meta.yaml" that is on your comment.

File: recipes/rhessys/build.sh
Comment (isuruf) [inline recipes/rhessys/build.sh:10]:
```suggestion
export INCLUDES='-I${PREFIX}/include'
```

File: recipes/rhessys/meta.yaml
Comment (isuruf) [inline recipes/rhessys/meta.yaml:20]:
```suggestion
```

File: recipes/rhessys/meta.yaml
Comment (isuruf) [inline recipes/rhessys/meta.yaml:26]:
```suggestion
```

File: recipes/rhessys/build.sh
Comment (isuruf) [inline recipes/rhessys/build.sh:17]:
```suggestion
mv $F_MASTER/rhessysecv${PKG_VERSION} $PREFIX/rhessysecv${PKG_VERSION}
```

File: recipes/rhessys/meta.yaml
Comment (isuruf) [inline recipes/rhessys/meta.yaml:30]:
```suggestion
    - rhessysecv{{ version }} -version 
```

**Comment by DavidChoi76** [comment]
@isurf Thank you for your suggestion. I edited every comment.

File: recipes/rhessys/meta.yaml
Comment (isuruf) [reply recipes/rhessys/meta.yaml:7]: See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L19

File: recipes/rhessys/meta.yaml
Comment (dhirschfeld) [inline recipes/rhessys/meta.yaml:10]:
```suggestion
  sha256: f7a7a26a977638fc66bd0a195fb90ca930cb956898d7baaeef249e8d311e39a0
```

**Comment by dhirschfeld** [comment]
```
RuntimeError: SHA256 mismatch: 'f7a7a26a977638fc66bd0a195fb90ca930cb956898d7baaeef249e8d311e39a0' != '6edf6582ca076b87c26551fa7a773d9e3de48c25'
```

File: recipes/rhessys/meta.yaml
Comment (DavidChoi76) [reply recipes/rhessys/meta.yaml:10]: @dhirschfeld Thank you for your suggestion, but where is this number (f7a7a26a977638fc66bd0a195fb90ca930cb956898d7baaeef249e8d311e39a0) from?
I set the URL, but even though the URL is exact, that occurred an error. So I put the sha256 number in meta.yaml.

**Comment by DavidChoi76** [comment]
This PR is ready for review again.

**Review by isuruf — CHANGES_REQUESTED** [review]
This code seems to be a fork of https://github.com/RHESSys/RHESSys with GPL license added. That's not going to work. You'll have to ask the original authors to license the code.

File: recipes/rhessys/meta.yaml
Comment (dhirschfeld) [reply recipes/rhessys/meta.yaml]: You download that binary and then calculate the sha256 checksum using whatever tools you have to hand:

image

**Comment by DavidChoi76** [comment]
This is EC(East Coast) version of RHESSys. So we named it "rhessysec". The version is branched from https://github.com/RHESSys/RHESSys version 5.20. But the current version is a little bit different. https://github.com/RHESSys/RHESSy is focusing on dry ecosystems and forest disturbances.
EC version is focusing on urban and LID analysis. @lband is the original author of this model. http://fiesta.bren.ucsb.edu/~rhessys/setup/downloads/files/RHESSysTagueEA2004.pdf
We will discuss how to merge two RHESSys in the future. But it will take time. Before that, for convenience to use rhessysec version, we want to publish conda version of rhessysec.
We talked to https://github.com/RHESSys/RHESSy GitHub group about this process being carried out. 
Do you think do we still have to do something more with https://github.com/RHESSys/RHESSy GitHub group?

**Comment by isuruf** [comment]
> Do you think do we still have to do something more with https://github.com/RHESSys/RHESSy GitHub group?

Yes, since they are copyright holders of some parts of the code, we need their permission to distribute the code.

**Comment by DavidChoi76** [comment]
Ok, I will ask the original author of https://github.com/RHESSys/RHESSy. Thank you.

**Pull request closed without merging**
